source/Squadron.cpp

```diff
@@ -0,0 +1,32 @@
+/* Squadron.cpp
+Copyright (c) 2019 by Peter van der Meer
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+using namespace std;
+
+#include "DataNode.h"
+#include "Ship.h"
+#include "Squadron.h"
+#include <string>
+
```

Comment:
There should be an emtpy line between "" includes and <> includes in both header and cpp files.

using namespace std; should come after the includes.